packages/integration-tests/src/setup_test_directory.ts

```diff
@@ -1,20 +1,19 @@
 import { existsSync } from 'fs';
-import fs from 'fs/promises';
-import { fileURLToPath } from 'url';
+import fsp from 'fs/promises';
+import path from 'path';
+import * as os from 'os';
 
 /**
  * Root test directory.
  */
-export const rootTestDir = fileURLToPath(
-  new URL('./e2e-tests', import.meta.url)
-);
+export const rootTestDir = path.join(os.tmpdir(), 'e2e-tests');
```

Comment:
I have to say that I like when test project appear somewhere repo root (like it was before here).
The reason is that when I work on these tests locally I can inspect the output within the same IDE instance and I don't have to jump over to OS temp dir.

Perhaps common ground could be to keep `e2e-tests` where they are but perhaps add some path segment per test session.
I.e. `e2e-tests/<some_uuid>/<rest>` ?

Reply:
Btw. If we decide to follow suggestion to bootstrap test projects using `create-amplify` then perhaps we should make change as suggested and create folder outside of repo root.

Reply:
> then perhaps we should make change as suggested and create folder outside of repo root.
I'm confused. Did you suggest to create the test folder inside our outside of the repo root?

The reason why I changed it to a tmpDir was because if it's in the repo root, it shares the node_modules with the repo, which might hide some potential issues.

Reply:
I realized later that if we want to "install" or use "create amplify" then it might be better to have test projects outside of repo root because of what you mentioned.

Reply:
If we decide to use the repo root and share the node_modules, we should be able to remove the install commands in this PR.

Also, we may not need to use `npx-proxy`, so we don't have to make that change in the PR either.

Reply:
I'm in favor of leaving test dirs within the repo root and not doing the extra installation bits.

Reply:
on the other hand we should figure out what we want to test.
if we want to test that deployer works with "yarn resolved" workspace then perhaps going through full process makes some sense.
On the other hand it seems that we perhaps don't have to run every test project to assert that deployer works with each package manager.
So perhaps an option to consider is to have separate e2e "deployment test" that explores one deployment scenario but runs with multiple package managers.
Or maybe we should just add deployement step to create amplify tests and leave deployment tests as they are.